src/features/commands/server.js

```diff
@@ -51,7 +51,7 @@ Roles: **${serverData.roles.cache.size}**
                         { name: 'Boosters', value: `🚀 ${serverData.premiumSubscriptionCount}`, inline: false },
                         { name: 'Explicit content filtering', value: `${serverData.explicitContentFilter}`, inline: true },
                         { name: 'Verification level', value: `${serverData.verificationLevel}`, inline: true },
-                        { name: 'Features', value: `${serverData.features.length >= 1 ? serverData.features.map(feature => `⦿ ${capitalizeFirstLetter(feature.toLowerCase())}`) : "No features available."}`, inline: false },
+                        { name: 'Features', value: `${serverData.features.length >= 1 ? serverData.features.map(feature => `• ${capitalizeFirstLetter(feature.toLowerCase())}\n`) : "No features available."}`.replaceAll(",", ""), inline: false },
```

Comment:
This is resource intensive and I don't see why it should be done this way, refer to this instead https://github.com/Microsoft-community/Dotsimus/blob/main/src/features/commands/user.js#L40

Reply:
Didn't know about this, thanks!